components/interface/FocusTerm.js

```diff
@@ -194,12 +256,8 @@ export default class FocusTerm extends React.Component {
       var entity = Model[otherId];
       // clear query builder unless ctrl pressed them add to compound.
       this.props.queryBuilder.open();
-      if (!GEPPETTO.isKeyPressed("shift")) {
```

Comment:
@ddelpiano are we sure this is not needed anymore (the SHIFT check)?

Reply:
will fix this shortly

Reply:
Note: Revert on this still needed